src/game.cpp

```diff
@@ -1912,44 +1916,21 @@ bool Game::initSound()
 	return true;
 }
 
-bool Game::createSingleplayerServer(const std::string map_dir,
-		const SubgameSpec &gamespec, u16 port, std::string *address)
+bool Game::createServer(const std::string map_dir,
+		const SubgameSpec &gamespec,
+		const std::string &listen)
```

Comment:
Even if it's obvious that listen is an argument of the function in this context, we should still try to avoid shadowing symbols.